carparkmanager-persistence/src/main/java/cz/muni/fi/pa165/carparkmanager/persistence/dao/DriveDaoImpl.java

```diff
@@ -19,7 +19,7 @@
     @PersistenceContext
     private EntityManager em;
     
-    public static String SELECT_ALL = "SELECT d FROM Drive d";
+    public static final String SELECT_ALL = "SELECT drive FROM Drive drive";
```

Comment:
nemoze to byt "drive" - ty tomu vlastne vytvaras alias s menom "drive", ale tabulka "drive" v DB existuje.. takze to musi byt cokolvek ine okrem "drive" ... napriklad predtym to bolo dobre "SELECT d FROM Drive d";